### ISSUE TITLE
Not yet compatible to Terminus v2.0.0

### ISSUE THREAD

**Issue by carl-alberto** [pantheon-systems/terminus-acme-plugin#5]
Tried to install and run the command with Terminus 2.0.0

And getting:
```
[warning] Could not load plugin pantheon-systems/terminus-acme-plugin because it is not compatible with this version of Terminus.
Terminus 2.0.0

```

**Comment by amklose** [comment]
None of the terminus plugins I was using are compatible with 2.0 yet. I'll have to downgrade so I can use this one, as it's urgent.

**Comment by amklose** [comment]
@carl-alberto Just kidding! - I just saw that there is a [t2 branch](https://github.com/pantheon-systems/terminus-acme-plugin/tree/t2). You should be able to use that with Terminus 2.

To install, use the following commands:
`mkdir -p ~/.terminus/plugins`
`curl https://github.com/pantheon-systems/terminus-acme-plugin/archive/t2.tar.gz -L | tar -C ~/.terminus/plugins -xvz`

And then it looks like the readme file on the t2 branch says to run the command like this:
`terminus https:challenge:dns-txt example.com`
but I don't think the actual plugin code has been updated for that. The command still worked for me using the 1.x syntax:
`terminus alpha:https:challenge:dns-txt <site>.<env> example.com`

**Comment by carl-alberto** [comment]
Nice find @amklose :) Thanks!

**Comment by greg-1-anderson** [comment]
Fixed by #4

**Comment by carl-alberto** [comment]
Thanks @greg-1-anderson verified working after updating to Terminus 2.0.1 and acme plugin latest 1.x branch
